src/lib/services/data/schemas.ts

```diff
@@ -1,3 +1,3 @@
 import { z } from 'zod';
 
-export const getDataDashboardProjectByLabelSchema = z.string();
+export const getDataDashboardProjectByIdSchema = z.optional(z.string().cuid());
```

Comment:
Just to keep it across the schemes the same:
```suggestion
export const getDataDashboardProjectByIdSchema = z.string().cuid().optional();
```

Reply:
Done!